lib/find.ts

```diff
@@ -13,7 +14,7 @@ interface FileHandler {
 
 export async function find(
   src: string,
-  excluded: readonly Path[] = [],
+  excludePatterns: readonly string[] = [],
```

Comment:
Maybe we should test that find() behaves properly when excludePatterns is empty, or in other possible edge cases?

Reply:
(I just mean old tests, more than a week ago, calls find without second argument)